apps/io-services-cms-webapp/src/main.ts

```diff
@@ -76,6 +83,12 @@ const fsmPublicationClient = ServicePublication.getFsmClient(
   servicePublicationStore
 );
 
+const servicesContainer = cosmosdbClient
+  .database(config.COSMOSDB_NAME)
```

Comment:
it should be `config.LEGACY_COSMOSDB_NAME`, right?

Reply:
fix with fcc7de867df813842057b7ede03843a930a72ded